guides/puppet-web-apps/html-template-guidelines/README.md

```diff
@@ -0,0 +1,157 @@
+
+# HTML Template Guidelines
+
+In order to harness the full power of Starcounter, applications should be built to accomodate for complete visual and functional interoperability. To make this process easier for developers, we provide these guidelines for HTML templates which, when followed, will allow applications to achieve seamless visual integration with other applications.
+
+For applications that have previously been developed without following these guidelines, take a look at the article [Convert Existing HTML Templates to starcounter-include](https://starcounter.io/making-apps-blendable/). To get a technical background, read the article [Layout compositions for HTML partials](https://starcounter.io/layout-compositions-html-partials/).
+
+### Guideline 1: Separate Functionality and Layout
+
+To make applications look great when running independently while also allowing them to visually blend with other applications, it is beneficial to separate the functionality and layout in the HTML template. This is easily accomplished using the `<template is="starcounter-composition">` element.
+
+The basic boilerplate of a Starcounter HTML template, which is created by adding a `Starcounter HTML template with dom-bind` file in Visual Studio, looks like this:
+
+```html
+<link rel="import" href="/sys/polymer/polymer.html">
+
+<template>
+    <template is="dom-bind">
+
+    </template>
+</template>
+```
+
+To separate functionality and layout in this html file, the element mentioned above, `<template is="starcounter-composition">` should be used. This element should contain the layout of the application while the `<template is="dom-bind">` should contain the functional elements. In code, this is how it looks:

```

Comment:
Because this is a guideline for people who **want** to be told what to do. 

I think there should be some disclaimer once in the Docs - that we use Polymer in our Docs for reference, but other libraries can be used.